### PR TITLE
Log the error content from FxA

### PR DESCRIPTION
**Pull request by mstriemer**
Include the error message in the logger.

### CODE REVIEW

File: src/olympia/accounts/verify.py [hunk 1]
```diff
@@ -66,7 +67,10 @@ def get_fxa_profile(token, config):
...
     else:
-        log.info('Profile returned non-200 status {status} [{token}]'.format(
-            token=token, status=response.status_code))
+        log.info(
+            'Profile returned non-200 status {status} {body} '
+            '[{token}]'.format(
+                token=token, status=response.status_code,
+                body=response.content))
         raise IdentificationError('Could not find profile for {token}'.format(
             token=token))
```
Comment (andymckay) [inline src/olympia/accounts/verify.py]: is there anything scary in that content?

Comment (mstriemer) [inline src/olympia/accounts/verify.py]: It should just be a JSON payload but I'd rather not parse it and have it fail. It should just say what field is wrong not what values it has.

Messing with it locally:
```
Token returned non-200 status 400 {"code":400,"errno":109,"error":"Bad Request","message":"Invalid request parameter","info":"https://github.com/mozilla/fxa-oauth-server/blob/master/docs/api.md#errors","validation":{"source":"payload","keys":["client_secret"]}} [8310f22388887cf20000be2a8981a82e9ddf05cdcdfbc4ab96be4ef32037f779] 
Profile returned non-200 status 401 {"code":401,"errno":100,"error":"Bad Request","message":"Unauthorized","reason":"Invalid request parameter"} [be9dd9d782e9165bcd9fc4734fa9ac93eae81d57be4b5b831abe71f6710ea85efoo]

Comment (andymckay) [inline src/olympia/accounts/verify.py]: ok, r+

**Comment by kumar303** [comment]
Good idea. If it becomes too noisy we could move it to `log.debug()` but I'd say try it in `info()` first.

r+

**Pull request merged**